src/main/java/uiPages/ProductPage.java

```diff
@@ -0,0 +1,70 @@
+package uiPages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import helpers.WebElementHelper;
+
+import static helpers.WebElementHelper.areVisible;
+import static helpers.WebElementHelper.setFieldValue;
+
+public class ProductPage extends BasePage {
+
+    private static final String URL_PATH = "/product";
+
+    @FindBy(className = "product-essential__description")
+    private WebElement productDescriptionField;
+
+    @FindBy(className = "product-essential__specification")
+    private WebElement productSpecificationField;
+
+    @FindBy(xpath = "//button[@class='single_add_to_cart_button button alt disabled wc-variation-selection-needed']")
+    private WebElement disabledAddToCart;
+
+    @FindBy(xpath = "//*[@id=\'pa_color\']/option[2]")
+    private WebElement attributeColor;
+
+    @FindBy(xpath = "//*[@id=\'pa_size\']/option[2]")
+    private WebElement attributeSize;
```

Comment:
attributeColor and attributeSize are 'selector' html elements. 
There is an easy method to manipulate this kind on element: using the selenium 'Selector' class (https://www.guru99.com/select-option-dropdown-selenium-webdriver.html).
Please rewrite the implementation of color and size attributes using this method. 
If you use this method you can identify the color and size elements by Id.

Reply:
done